notes/stories/film/film-scifi-2020s.st.txt

```diff
@@ -285,6 +285,52 @@ unethical business practices [It turned out that the big pharma company that'd e
 speculative genetic engineering [It was mentioned how a pharmaceutical billionaire had genetically engineered a bull shark to be intelligent.]
 
 
+movie: Tenet (2020)
+===================
+
+:: Title
+Tenet
+
+:: Description
+Tenet is a 2020 science fiction action thriller film written and directed by
+Christopher Nolan, who produced it with Emma Thomas.
+
+Synopsis: The film follows a secret agent who learns to manipulate the flow of
+time to prevent an attack from the future that threatens to annihilate the
+present world.
+
+:: Date
+2020-08-26
+
+:: References
+https://en.wikipedia.org/wiki/Tenet_(film)
+
+:: Ratings
+1 <paul>
+
+:: Major Themes
+time travel [A main novelty of the film is that people travel in time by reversing the entropy of things and people.]
+reversed time [Central to the plot was the Protagonist, and others, making things and people move in time by somehow reversing entropy.]
+husband and wife [One storyline follows Kat as she attempts to stop her estranged husband, Sator, from blowing up the world. The Protagonist interacted with the arms dealer Priya and her husband Sanjay.]
+anthropogenic existential risk to civilization [It was up to the Tenet organization to stop the Russian Oligarch Sator from taking out the whole world with him before he died of inoperable cancer.]
```

Comment:
Is the oligarch dying from cancer?
Could we explain the mode by which the world would be taken out?

Reply:
Updated the motivation. Does any appropriate specialization of **anthropogenic existential risk to civilization** come to mind?

Also I slipped in **time travel paradox** since we have it now.